### ISSUE TITLE
Adjust modal content to make opening links in new tab more accessible.

### ISSUE THREAD

**Issue by wrenaria** [ucd-library/aggie-experts#543]
Some modals open content in a new tab without warning the user, which is an accessibility issue for vison-impaired/screenreader users. In general, it is best practice to have links open in the same tab when possible to avoid disorienting the users.

Accessibility Reference: https://www.w3.org/TR/WCAG20-TECHS/G200.html

For better accessibility compliance, we need to do one or more of the following:
1. Do not open links in a new tab.
2. Warn users using text associated with the link itself.
3. Include the warning in the modal text prior to the yellow "edit" button that appears on most modals.

Vessela would prefer to maintain links opening in new tabs and incorporate option 3. to this end, I recommend that we do a mix where possible. 

# Error modal:
For Error screens, which do not have dedicated yellow "edit" buttons, the warning should be in-line as part of the link text.

Screenshot from Figma:
![Screenshot 2024-07-01 at 12 11 20 PM](https://github.com/ucd-library/aggie-experts/assets/17831969/f80db344-229c-4afe-9cfe-fd599c67ecaa)

# Standard editing modal:
For standard editing modals, alter the text prior to the button to say `You will be redirected to this system in a new tab.`

In addition, we should also make sure to include screen-reader only aria labels that say `edit [item] in a new tab` for the the yellow "Edit" buttons for the standard modals. For the example below, the aria label would be `edit introduction in a new tab` (@UcDust I believe this to be the correct approach for this, please correct me if I am wrong).

Screenshot from Figma: 
![Screenshot 2024-07-01 at 11 46 37 AM](https://github.com/ucd-library/aggie-experts/assets/17831969/2aad4507-65aa-4a0e-bce0-99c7e4cd2439)

**Comment by Vensberg** [comment]
Looks good.

**Comment by UcDust** [comment]
@Vensberg @wrenaria 

Currently the _Website_ links have a modal like so-
<img width="723" alt="Screenshot 2024-07-09 at 10 34 41" src="https://github.com/ucd-library/aggie-experts/assets/103536473/76300466-28bf-4e4e-b83e-476c6fc1ed7b">

However the _Roles_ just open directly to ODR. I'm guessing we'll want a modal for Roles now indicating it will open in a new tab, but what would you like the text of that modal to be?

**Comment by Vensberg** [comment]
How about "Academic roles and titles are managed via the UC Davis Online Directory. You will be redirected to this system in a new tab."?

**Comment by UcDust** [comment]
Great, thanks!
